addons/api/tests/unit/models/target-test.js

```diff
@@ -249,11 +249,6 @@ module('Unit | Model | target', function (hooks) {
       2,
       'Target has two entires in brokered_credential_source_ids'
     );
-    assert.strictEqual(
```

Comment:
Why are we removing this?

Reply:
The field is now a simple array rather than a fragment array, therefore `credentialSources` which depends on it is no longer recomputed when the IDs array changes.  This computed field is recomputed only on a model reload/refresh/rollback.  So this initial zero length check computes `credentialSources`, but later checks would not result in recomputation.  Arguably this might be avoided by resetting the array to its current value (we've done this elsewhere).  I can make that update.